exercises/chat/online_examples/pairclient.py

```diff
@@ -1,18 +1,17 @@
-#http://learning-0mq-with-pyzmq.readthedocs.org/en/latest/pyzmq/patterns/pair.html
+# http://learning-0mq-with-pyzmq.readthedocs.org/en/latest/pyzmq/patterns/pair.html
 
-import zmq
-import random
-import sys
 import time
 
+import zmq
+
 port = "5556"
 context = zmq.Context()
 socket = context.socket(zmq.PAIR)
 socket.connect("tcp://localhost:%s" % port)
```

Comment:
Can we replace all of this old string formatting with the new `str.format()`? It is to be preferred in Python 3 over the old-style `%` operator. See [the documentation](https://docs.python.org/3/library/stdtypes.html?highlight=sprintf#printf-style-string-formatting).

So this line would become 
```suggestion
socket.connect("tcp://localhost:{}".format(port))
```